<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="save_path">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="target_path">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>작업폴더</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>출력폴더</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_target_path">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>선택</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save_path">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>선택</string>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="fileList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>751</width>
      <height>361</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left(시작)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right(끝)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TOP(시작)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bottom(끝)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>(1) 양평군체 L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="right_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>(1) 양평군체 L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="top_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>(1) 양평군체 L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="bottom_edit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>(1) 양평군체 L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>580</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>자르기 시작</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
